marconi/src/Marconi/CardanoAPI.hs

```diff
@@ -0,0 +1,47 @@
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE PatternSynonyms    #-}
+{-# LANGUAGE TypeFamilies       #-}
+-- | This module provides several type aliases and utility functions to deal with the current Cardano era.
+module Marconi.CardanoAPI
+       (
+       CurrentEra,
+       pattern AsCurrentEra,
+       C.TxIn(C.TxIn),
```

Comment:
I think this isn't worth it: exporting TxIn from here and using it imported Marconi.CardanoAPI in other modules will give rise to the question of how is it different from the one in Cardano.Api, and the answer is that there are no differences. Maybe when we get the package `cardano-api-extras` that will serve as a front for cardano-api that we directly control and which' contents we want to upstream, it would make sense to re-export Cardano.Api types from there, but otherwise I think it's better to use things from their original location as it builds upon knowledge on having used those modules earlier.

Reply:
I hesitated to do so for that exact reason. The pros were that we centralised our dependencies on cardano-api, whether or not these types are era-dependent. But you're right, even if reexporting is a common practice, it probably doesn't worth the effort here.

Reply:
Reexporting is fine for exporting definitions from modules that are otherwise internal, or to provide a nice API from a single location. But Cardano.Api is not internal, and it already is _the_ entrypoint to what cardano-api has to offer; and also as Marconi.CardanoAPI doesn't export all of Cardano.Api then most of the time Cardano.Api will already be imported, so having one type go through another module raises the question of why - does it do anything different? - but it doesn't, and it isn't era-dependent either.